sources/Runner/Runner.cpp

```diff
@@ -67,13 +67,12 @@ namespace BBM
 	{
 		auto scene = std::make_shared<WAL::Scene>();
 		RAY3D::Cube cube(Vector3f(-5, 0, -5), Vector3f(3, 3, 3), RED);
```

Comment:
You should instantiate the cube directly in the addComponent method, doing this here duplicate the resource unnecessarily (and we don't really want the raylib to create too many things)